OfflineInstallPreparation.ps1

```diff
@@ -81,6 +81,14 @@ if ($Signature.Status -eq 'Valid' -and $Signature.SignerCertificate.Subject -eq
 }
 
 # Initialize environment, ensure Chocolatey For Business, etc.
+
+#Check for and configure FIPs enforcement, if required.
```

Comment:
Is there a reason this is in the OfflineInstallPrep step, which may be run on a random machine (to offline-prep), and not in one of the server-only scripts?

Reply:
I agree with @JPRuskin here. I think we should just do this on the server itself in Start-C4bSetup.ps1 instead of the offline script, so it only affects the server we stand up and not the potential download machine in a truly offline setup scenario.

This needs to be tested to see if a package download fails if FIPS is not set beforehand.

Reply:
We run the OfflineSetup script _regardless_. It's either ran independently by a user, or automatically by our Start-C4bSetup script. In _either_ case, FIPs is only enabled in Chocolatey if the machine the script is running against is FIPs enabled. And if FIPs is enabled, we will want to ensure Chocolatey is properly configured for it, regardless of context.

Reply:
Though I agree that it should be done somewhere, it's still an odd look in my eyes to apply configuration in the offline prep script that currently doesn't do any configuration. Worth noting that download doesn't do any checksumming, so probably doesn't need the setting (or, at least, it seems to work on my box). It's a shame that [this](https://github.com/chocolatey/choco/issues/2590) hasn't been worked yet.

Overall, it's a good thing to check on any box that's running our code - so I suggest, instead, we move this check to the module (ClientSetup needs it separately). That way it'll be run on any FIPS-compliant environment regardless of which script we start with. I'll get that sorted.